app/views/stats/usage/index.html.slim

```diff
@@ -27,7 +27,7 @@ javascript:
         methodsTableContainer: '.StatsMethodsTable-container',
         csvLinkContainer: '.StatsCSVLink-container',
         usageMetricsUrl: '#{admin_service_stats_usage_path(@service, format: :json)}',
-        hitsMetricName: '#{@metrics.find(&:hits?).system_name}',
+        hitsMetricName: '#{@service.metrics.where(system_name: "hits").last.system_name}',
```

Comment:
This will perform an extra query to the DB (`@service.metrics`) that can be avoided since the `@metrics` instance variable already holds all the metrics we need for this view.

To avoid the extra query, we could search the values we need from `@metrics` instead. Something like:
```suggestion
        hitsMetricName: "#{@metrics.find { |metric| metric.hits? && metric.owner_type == 'Service' }.system_name}",
```

Maybe even create a `Metric#service?` or `Metric#owner_type_service?` method that checks if `owner_type == 'Service'` so we may do something like:
```suggestion
        hitsMetricName: '#{@metrics.find { |metric| metric.hits? && metric.service? }.system_name}',
```